wielder/wield/project.py

```diff
@@ -267,6 +268,39 @@ def get_local_path(conf, relative_path, bucket_name=None):
     return local_destination
 
 
+def configure_project_module(conf, module, app):
+    module_config = conf[module]
+    if module_config:
```

Comment:
this fails if the key is not in the conf object, you need to check for existence